client/tests/e2e/specs/test_rename_workspace_modal.ts

```diff
@@ -0,0 +1,45 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```

Comment:
You can just add the tests to `test_workspaces_page` (https://github.com/Scille/parsec-cloud/blob/master/client/tests/e2e/specs/test_workspaces_page.ts).